packages/office-ui-fabric-react/src/components/Fabric/Fabric.styles.ts

```diff
@@ -21,7 +21,8 @@ export const getStyles = memoizeFunction((
     root: mergeStyles([
       theme.fonts.medium,
       {
-        color: theme.palette.neutralPrimary,
+        color: theme.semanticColors.bodyText,
+        '&.ms-Fabric': { color: theme.semanticColors.bodyText }, // overwrite default ms-Fabric's color with one that themes
```

Comment:
this change doesn't make sense to me.

color is already assigned to this class. In addition, you're creating a selector (which is 2 levels of specificity deep) setting the color again.

What is the purpose of this?